docs/guides/segment-migration-guide.md

```diff
@@ -0,0 +1,300 @@
+| write_key | api_key   | Unique key to validate source of the data. |
+| Workspace | Project   | [Projects](https://help.amplitude.com/hc/en-us/articles/360058073772-Create-and-manage-organizations-and-projects#create-a-project) allow you to organize your data.  |
+| User      | User      | User who is performing action.             |
+| Identify  | Identify  | [Identify](/analytics/what-is-amplitude/#user-properties-are-details-about-your-user) updates properties/attributes of the user.|
+| Track     | Event     | [Event](/analytics/apis/http-v2-api-quickstart/) in Amplitude tracks the action user is performing.|
+| Screen    | Event     | Create an Event to track Screen views.|
+| Page      | Event     | Create an Event to track Page views.|
+| Group     | Group     | [Group](/guides/accounts-instrumentation-guide/) is a collection of users. In Amplitude one user could belong to multiple groups. Each group can have properties/attributes that will be available to query/forward on actions performed by any user in the group.|
+| Plugins   | Plugins   | [Plugins](/data/ampli/plugin/) les you extend Amplitude by running a sequence of custom code on every event.|
+
+
+=== "Browser"
+
+    Documentation for [Browser Typescript SDK](/data/sdks/typescript-browser/).
+
+    <table>
+    <tr>
+    <td>  </td> <td> Segment </td> <td> Amplitude </td>
+    </tr>
+    <tr>
+    <td> Identify </td>
+    <td> 
+    ```typescript
+    analytics.identify('12091906-01011992', {
+        name: 'Grace Hopper',
+        email: 'grace@usnavy.gov'
+    });
+    ```     
+    </td>
+    <td>
+    ```typescript
+    setUserId('12091906-01011992');
+    const identifyObj = new Identify();
+    identifyObj.set('name', 'Grace Hopper');
+    identifyObj.set('email', 'grace@usnavy.gov');
+    identify(identifyObj);
+    ```
+    </td>
+    </tr>
+    <tr>
+    <td> Track </td>
+    <td>
+    ```typescript
+    analytics.track('Article Completed', {
+        title: 'How to Create a Tracking Plan',
+        course: 'Intro to Analytics',
+    });
+    ``` 
+    </td>
+    <td>
+    ```typescript
+    track('Article Completed', {
+        title: 'How to Create a Tracking Plan',
+        course: 'Intro to Analytics',
+    });
+    ``` 
+    </td>
+    </tr>
+    <tr>
+    <td> Group </td>
+    <td>
+    ```typescript
+    analytics.group('UNIVAC Working Group', {
+    principles: ['Eckert', 'Mauchly'],
+    site: 'Eckert–Mauchly Computer Corporation',
+    statedGoals: 'Develop the first commercial computer',
+    industry: 'Technology'
+    });
+    ```
+    </td>
+    <td>
+    ```typescript
+    setGroup('Working Group', 'UNIVAC');
+    const groupIdentifyObj = new Identify()
+    groupIdentifyObj.set('principles', ['Eckert', 'Mauchly']);
+    groupIdentifyObj.set('site', 'Eckert–Mauchly Computer Corporation');
+    groupIdentifyObj.set('statedGoals', 'Develop the first commercial computer');
+    groupIdentifyObj.set('industry', 'Technology');
```

Comment:
This seeming clunky compared to Segment. I feel chaining the set operations could make it more concise.
```suggestion
    groupIdentifyObj.set('principles', ['Eckert', 'Mauchly']);
     .set('site', 'Eckert–Mauchly Computer Corporation');
     .set('statedGoals', 'Develop the first commercial computer');
     .set('industry', 'Technology');
```